src/main.cpp

```diff
@@ -5884,13 +5884,13 @@ bool ProcessMessage(CNode* pfrom, std::string strCommand, CDataStream& vRecv, in
     else if (strCommand == NetMsgType::XPEDITEDBLK)
     {
         // ignore the expedited message unless we are at the chain tip...
-        if (!fImporting && !fReindex && IsChainNearlySyncd())
+        if (!fImporting && !fReindex && !IsInitialBlockDownload() && IsThinBlocksEnabled())
```

Comment:
For reviewers - I changed this because it's now the same condition as the xthin block code path.  I'm unsure which one is preferable but am fairly sure they should be the same.